agents-docs/quick-reference.md

```
@@ -56,12 +59,14 @@ cargo build --bin csm --features cli --quiet
 ```
 
 ## ADR Registry ↔ Disk Parity (ADR-0076)
+
 ```bash
 ./scripts/check-adr-parity.sh                 # errors on missing, warns on orphans
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)